src/Vts/Vts.csproj

```diff
@@ -133,4 +133,9 @@
     <PackageReference Include="System.Reactive" Version="5.0.0" />
     <PackageReference Include="System.Reflection" Version="4.3.0" />
   </ItemGroup>
+  <ItemGroup>
+    <Reference Include="System.Windows.Forms">
+      <HintPath>..\..\..\..\..\..\..\..\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.8\System.Windows.Forms.dll</HintPath>

```

Comment:
Did you add this reference by mistake? We should not have any hard-coded dlls

Reply:
Not sure how that got there.  Can I undo here using GitHub?

Reply:
You might be able to remove it in GitHub but it would be better to remove it locally and commit to your branch. You can probably  just remove it in the csproj file.

Reply:
Okay will do.